### PR TITLE
Add canvas recording

### PR DESCRIPTION
**Pull request by c298lee**
Add experimental canvas recording for replays. Using yarn 4.0.2

### CODE REVIEW

**Comment by c298lee** [comment]
> @c298lee please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @dotnet-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @dotnet-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @dotnet-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@dotnet-policy-service agree

File: src/NuGetTrends.Web/Portal/package.json [hunk 1]
```diff
@@ -23,8 +23,9 @@
     "@angular/platform-browser-dynamic": "^15.1.5",
     "@angular/router": "^15.1.5",
     "@sentry-internal/feedback": "^0.0.1-alpha.17",
-    "@sentry/angular-ivy": "^7.84.0",
-    "@sentry/integrations": "^7.84.0",
+    "@sentry-internal/rrweb": "2.3.0",
```
Comment (billyvg) [inline src/NuGetTrends.Web/Portal/package.json]: 2.5.0 is latest:

```suggestion
    "@sentry-internal/rrweb": "2.5.0",
```

File: src/NuGetTrends.Web/Portal/.yarnrc.yml [hunk 2]
```diff
@@ -1 +1,5 @@
+compressionLevel: mixed
+
+enableGlobalCache: false
+
```
Comment (billyvg) [inline src/NuGetTrends.Web/Portal/.yarnrc.yml]: Is this needed?

Comment (c298lee) [reply src/NuGetTrends.Web/Portal/.yarnrc.yml]: oh don't think so, lemme remove it

**Review by bruno-garcia — APPROVED** [review]
Thanks Catherine! I wonder what's up with yarn, CI is green though so all good

**Pull request merged**